packages/quantic/force-app/main/default/lwc/quanticTab/e2e/quanticTab.e2e.ts

```diff
@@ -0,0 +1,87 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {
+  useCaseEnum,
+  useCaseTestCases,
+} from '../../../../../../playwright/utils/useCase';
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+const expectedActionCause = 'interfaceChange';
+const expectedOriginContext = 'Search';
+const expectedTabsLabels = ['All', 'Case', 'Knowledge'];
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic tab ${useCase.label}`, () => {
+    test.describe('when clicking on a tab', () => {
+      test('should trigger a new search and send the correct UA analytics event', async ({
+        tab,
+        search,
+      }) => {
+        const selectedTabIndex = 0;
+        const searchResponsePromise = search.waitForSearchResponse();
+        const uaRequestPromise = tab.waitForTabSelectUaAnalytics();
+        await tab.clickTabButton(selectedTabIndex);
+
+        const searchResponse = await searchResponsePromise;
+        const searchResponseBody = searchResponse.request().postDataJSON();
+
+        expect(searchResponseBody).toEqual(
+          expect.objectContaining({
+            analytics: expect.objectContaining({
+              actionCause: expectedActionCause,
+              originContext: expectedOriginContext,
+            }),
+            tab: await tab.tabLabel(selectedTabIndex),
+          })
+        );
```

Comment:
This logic could be a method that you can call call from your page object.

Reply:
I will actually use the same pattern you used in the pager for this